samples/openthread/coprocessor/sample.yaml

```diff
@@ -1,33 +1,46 @@
 sample:
   description: Test OpenThread Co-Processor
   name: OpenThread Co-Processor sample
+
 tests:
   sample.openthread.coprocessor:
     build_only: true
-    extra_args: SNIPPET="ci;logging;vendor_hook"
+    tags: ci_build
+    platform_allow: >
+      nrf52833dk_nrf52833
+      nrf52840dk_nrf52840
+      nrf52840dongle_nrf52840
+      nrf21540dk_nrf52840
+    extra_args: >
+      SNIPPET="ci;logging;vendor_hook"
     integration_platforms:
-      - nrf52840dk_nrf52840
-      - nrf52840dongle_nrf52840
       - nrf52833dk_nrf52833
+      - nrf52840dk_nrf52840
       - nrf21540dk_nrf52840
-    platform_allow: >
-      nrf52840dk_nrf52840 nrf52840dongle_nrf52840 nrf52833dk_nrf52833 nrf21540dk_nrf52840
-    tags: ci_build
+      - nrf52840dongle_nrf52840
+
   sample.openthread.coprocessor.usb:
     build_only: true
-    extra_args: SNIPPET="ci;usb"
+    tags: ci_build
+    platform_allow: >
+      nrf52833dk_nrf52833
+      nrf52840dk_nrf52840
+      nrf21540dk_nrf52840
+    extra_args: >
+      SNIPPET="ci;usb"
+      CONFIG_OPENTHREAD_DIAG=y
```

Comment:
Why is this config needed here? Could it be added to `ci` snippet?

Reply:
Thanks for catching that, I left it by mistake.